src/components/application_manager/test/commands/mobile/set_audio_streaming_indicator_test.cc

```diff
@@ -494,18 +494,14 @@ TEST_F(SetAudioStreamingIndicatorRequestTest,
 
   ApplicationSharedPtr mock_app_empty;
   EXPECT_CALL(app_mngr_, application(_))
-      .Times(2)
+      .Times(1)
       .WillRepeatedly(Return(mock_app_empty));
 
   EXPECT_CALL(*mock_app_, RemoveIndicatorWaitForResponse(_)).Times(0);
   EXPECT_CALL(
       mock_message_helper_,
       CreateNegativeResponse(_, _, _, am::mobile_api::Result::GENERIC_ERROR))
```

Comment:
@VVeremjova fixed in https://github.com/smartdevicelink/sdl_core/pull/1368/commits/e1e998ffc82c12f956a5d10777d9061aa13daae2